tests/test_async.cpp

```diff
@@ -88,8 +88,8 @@ TEST_CASE("async periodic flush", "[async]")
     auto logger = spdlog::create_async<spdlog::sinks::test_sink_mt>("as");
     auto test_sink = std::static_pointer_cast<spdlog::sinks::test_sink_mt>(logger->sinks()[0]);
 
-    spdlog::flush_every(std::chrono::seconds(1));
-    std::this_thread::sleep_for(std::chrono::milliseconds(1700));
+    spdlog::flush_every(std::chrono::milliseconds (100));
+    std::this_thread::sleep_for(std::chrono::milliseconds(170));
```

Comment:
We are experiencing problems with sleep accuracy when the tests are running in vm. Please revert this test to the original 1 sec + 1700 ms